### DISCUSSION TITLE
text and lines padding.

### DISCUSSION THREAD

**Discussion by noone1235** [plantuml/plantuml#1835] [General]
![plantuml](https://github.com/plantuml/plantuml/assets/64534011/3d8a3114-6fa9-4288-9082-4b654d2a7be6)

How can i increase the distance between lines and avoid overwriting of text in here, Have tried to increase all kinds of padding , but it does not effect lines and text overlap?

**Comment by The-Lum** [comment]
Hello @noone1235,

Thanks for the report.
- What are your parameters?
_[]Hints:  `lineType ortho` + Plantuml GraphViz layout engine]_

## Some _[similar]_ images
Then here are some similar examples:

```puml
skinparam lineType ortho

Class "DateTime" as T

A --> B : Long long long label\n AB

A --> T : Long long long label\n AT
B --> T : Long long long label\n BT
C --> T : Long long long label\n CT
D --> T : Long long long label\n DT
E --> T : Long long long label\n ET
F --> T : Long long long label\n FT
G --> T : Long long long label\n GT
H --> T : Long long long label\n HT
I --> T : Long long long label\n IT

Z -l> T
```
>[![](https://www.planttext.com/api/plantuml/svg/XT2j3i8m68LXVwxYPButW21Ps_qbo4yHJ4aQQEYwPPtXxfaXE3xpgaUzzPNA6hOmCwSIvRr6Bjl-N02NGwriVDYZf3as39K2j3JcJCiJBqjvCFyIxZ7V2blxHV872QnchC1fnWcyPhoWqqmdw3NJ2mRD389HCwDWqinqdBxIv0Dz0000)](https://www.planttext.com/?text=XT2j3i8m68LXVwxYPButW21Ps_qbo4yHJ4aQQEYwPPtXxfaXE3xpgaUzzPNA6hOmCwSIvRr6Bjl-N02NGwriVDYZf3as39K2j3JcJCiJBqjvCFyIxZ7V2blxHV872QnchC1fnWcyPhoWqqmdw3NJ2mRD389HCwDWqinqdBxIv0Dz0000) 


```puml
skinparam lineType ortho

Class "DateTime\n" as T

A --> B : Long long long label\n AB

A --> T : Long long long label\n AT
B --> T : Long long long label\n BT
C --> T : Long long long label\n CT
D --> T : Long long long label\n DT
E --> T : Long long long label\n ET
F --> T : Long long long label\n FT
G --> T : Long long long label\n GT
H --> T : Long long long label\n HT
I --> T : Long long long label\n IT

Z -l> T
```
>[![](https://www.planttext.com/api/plantuml/svg/XT0x3i8m44NHVbOnIkyDK235TdvIobUXD8DaWOLZHt4QTayQHCSqjphLhQ-ODzbbvHHpm7iBNFRZMOXSabgvyN84n3KikM6f3AAMZRcovGlF9Jyu_IBta9RChVqY_44WgnaBSffn8AyP3-eqqu5wpVIWGJC3QDJC29eqCvsdRspIYJu0)](https://www.planttext.com/?text=XT0x3i8m44NHVbOnIkyDK235TdvIobUXD8DaWOLZHt4QTayQHCSqjphLhQ-ODzbbvHHpm7iBNFRZMOXSabgvyN84n3KikM6f3AAMZRcovGlF9Jyu_IBta9RChVqY_44WgnaBSffn8AyP3-eqqu5wpVIWGJC3QDJC29eqCvsdRspIYJu0) 

```puml
skinparam lineType ortho

Class "DateTime\n\t\t\t" as T

A --> B : Long long long label\n AB

A --> T : Long long long label\n AT
B --> T : Long long long label\n BT
C --> T : Long long long label\n CT
D --> T : Long long long label\n DT
E --> T : Long long long label\n ET
F --> T : Long long long label\n FT
G --> T : Long long long label\n GT
H --> T : Long long long label\n HT
I --> T : Long long long label\n IT

Z -l> T
```
>[![](https://www.planttext.com/api/plantuml/svg/XT2z2i8m5CNn_Jx5eNjUm45eadw1uvqaIuIWmJGfJHRVtZg8cvS3_-atdVgCUVExNv5Y3lpQ0ihU7eN895yhEkjRuBW6brtxh8ElOA8UIfsXSSAbv3lIB_uMailezHVn7yIa9QEPZ6GCavMCPHeaCp2DaXcP9ibCJBDaPgP5Cilnz1Kg7UWD)](https://www.planttext.com/?text=XT2z2i8m5CNn_Jx5eNjUm45eadw1uvqaIuIWmJGfJHRVtZg8cvS3_-atdVgCUVExNv5Y3lpQ0ihU7eN895yhEkjRuBW6brtxh8ElOA8UIfsXSSAbv3lIB_uMailezHVn7yIa9QEPZ6GCavMCPHeaCp2DaXcP9ibCJBDaPgP5Cilnz1Kg7UWD) 

@Plantuml Team:
_If that can help for debugging..._

Ref. and hints _(remember that `lineType ortho` is not so managed... with GraphViz...)_
- https://github.com/plantuml/backlog/issues/11
- https://forum.plantuml.net/search?q=linetype+ortho

Regards,
Th.
